src/people.js

```diff
@@ -15,17 +15,69 @@
  * @property {Person} tallerPerson - Taller person information
  */
 
-/** 
- * The height in cm of several people was taken, in addition, the age of each person and 
- * extra information are also known. An algorithm is required to calculate average age and 
- * average height. In addition, we need to know the information of the tallest person, and 
+/**
+ * The height in cm of several people was taken, in addition, the age of each person and
+ * extra information are also known. An algorithm is required to calculate average age and
+ * average height. In addition, we need to know the information of the tallest person, and
  * the information of the youngest person.
- * 
+ *
  * @param {Person[]} people - People information to be processed
- * 
+ *
  * @returns {PeopleResponse}  - Processed information
-*/
+ */
+
+function processNextPerson(people) {
+  let i = 0;
+  let tallerPerson = null;
+  let youngerPerson = null;
+  let ageSum = 0;
+  let heightSum = 0;
+
+  function processPerson(person) {
+    let personToProcess = { ...person };
```

Comment:
why you dont use the person object directly? btw, In the case it is necessary, it should be const